src/app/pages/Aparatos/AparatoDetails/index.js

```diff
@@ -0,0 +1,167 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Text from 'app/components/ui/Text';
+import Icon from 'app/components/ui/Icon';
+import Spacer from 'app/components/ui/Spacer';
+import UserInfoListElement from 'app/components/ui/UserInfoListElement';
+import SubMenu from 'app/components/ui/SubMenu';
+import PlainLink from 'app/components/ui/PlainLink';
+
+class AparatoDetails extends Component {
+  constructor(props) {
+    super(props);
+    const { history } = props;
+    this.aviso = this.props.aviso;
+    if (!this.aviso) history.push('/avisos');
+    this.device = this.getDevice();
+    if (!this.device) history.push('/avisos');
+  }
+  
+  getDevice = () => {
+    const { aviso, match } = this.props;
+    return aviso && aviso.devices[parseInt(match.params.id, 10)];
+  }
+
+  render(){
+    const { aviso } = this.props;
+    const device = this.device;
+    if (!aviso && !device) return null;
+    return(
+      <React.Fragment>
+          <SubMenu
+            activeStep={1}
+            title="DATOS DE APARATO/INTERVENCIÓN"
+            date={aviso.date}
+            time={aviso.time}
+            actions={
+              <React.Fragment>
+                <PlainLink 
+                  to={"/aparato/details/"+this.props.match.params.id+"/history"}
```

Comment:
mejor string literals
```suggestion
                  to={`/aparato/details/${this.props.match.params.id}/history`}
```